flavors/web-app/src/main/java/cf/dmms/app/web/api/book/BookUploadController.java

```diff
@@ -42,6 +43,7 @@ public BookUploadController(BookService bookService, BookMapper bookMapper, Obje
 
 	@PostMapping
 	public BookDto addBook(
```

Comment:
I think that this endpoint could be just in book controller. Consider moving part of methods to BookService or create separate service for uploading